Arg, linty linterson!

The `flake8` env should be added to the [default list][1] in `tox.ini`. If so, I'd have caught the [failure][2] locally

```
./tests/test_client.py:911:80: E501 line too long (80 > 79 characters)
./tests/test_client.py:912:80: E501 line too long (80 > 79 characters)
./tests/test_client.py:914:80: E501 line too long (80 > 79 characters)
./tests/test_client.py:915:80: E501 line too long (80 > 79 characters)
./tests/test_client.py:917:80: E501 line too long (80 > 79 characters)
```

[1]: https://github.com/google/oauth2client/blob/c456547a8ad6058a6a3322940c28ecf5398f5ade/tox.ini#L2
[2]: https://travis-ci.org/google/oauth2client/jobs/149338896

--- a/tests/test_client.py
+++ b/tests/test_client.py
@@ -908,14 +908,14 @@ class BasicCredentialsTests(unittest2.TestCase):
             self.assertEqual(token_response, self.credentials.token_response)
 
     def test_recursive_authorize(self):
-        # Tests that OAuth2Credentials doesn't introduce new method constraints.
-        # Formerly, OAuth2Credentials.authorize monkeypatched the request method
-        # of the passed in HTTP object with a wrapper annotated with
-        # @util.positional(1). Since the original method has no such annotation,
-        # that meant that the wrapper was violating the contract of the original
-        # method by adding a new requirement to it. And in fact the wrapper
-        # itself doesn't even respect that requirement. So before the removal of
-        # the annotation, this test would fail.
+        # Tests that OAuth2Credentials doesn't introduce new method
+        # constraints. Formerly, OAuth2Credentials.authorize monkeypatched the
+        # request method of the passed in HTTP object with a wrapper annotated
+        # with @util.positional(1). Since the original method has no such
+        # annotation, that meant that the wrapper was violating the contract of
+        # the original method by adding a new requirement to it. And in fact
+        # the wrapper itself doesn't even respect that requirement. So before
+        # the removal of the annotation, this test would fail.
         token_response = {'access_token': '1/3w', 'expires_in': 3600}
         encoded_response = json.dumps(token_response).encode('utf-8')
         http = http_mock.HttpMock(data=encoded_response)

